frontend/app/src/views/ImportData.vue

```diff
@@ -56,12 +68,60 @@
         </v-row>
         <v-row>
           <v-col cols="12">
-            <v-img
-              max-width="200"
-              class="import-data__cointracking-info elevation-1"
-              :src="require('../assets/images/cointracking_info.png')"
-              @click="coinTrackingInfoImport()"
-            ></v-img>
+            <div
+              class="import-data__crypto-com elevation-1"
+              @click="importData('crypto.com')"
+            >
+              <v-img
+                max-width="200"
+                :src="require('../assets/images/import/crypto_com.png')"
+              ></v-img>
+            </div>
+          </v-col>
+        </v-row>
+        <v-row>
+          <v-col cols="12">
+            Important notes for importing data from
+            <strong>crypto.com mobile app</strong>'s CSV exports.
+            <ul>
+              <li>
+                [Early stage] This import script may miss some transactions
+                types, like withdrawals. Since we can't have an exhaustive list
+                of that types, we develop it using what we knows at the time. If
+                you think that import script can be improved and you can help
+                providing more data type,
+                <external-link
+                  url="https://github.com/rotki/rotki/issues/new/choose"
+                >
+                  please let us know
+                </external-link>
+                .
+              </li>
+              <li>
+                <strong>
+                  It only concerns the Crypto.com mobile application.
+                </strong>
+                If you want to connect your Crypto.com Exchange account, please
+                <router-link to="/accounts-balance/exchange-balances">
+                  connect it as an exchance
+                </router-link>
+                since there is an API.
```

Comment:
Should we have this if here? We don't yet support crypto.com API. Perhaps for now change it to:

"please wait until we support it and then connect to it as an exchange"